src/Compilers/CSharp/Portable/BoundTree/BoundExpression.cs

```diff
@@ -238,6 +239,29 @@ public override Symbol ExpressionSymbol
                 return this.Method;
             }
         }
+
+        public Location? InterceptableLocation
```

Comment:
Considered moving this property on `BoundCall`. It seems only applicable to calls, not to any expression. 
Our usage seems to work with that (we only try to intercept when lowering `BoundCall` nodes).